cider/exec/nextgen/jitlib/base/ValueTypes.h

```diff
@@ -47,6 +60,16 @@ struct JITTypeTraits<VOID> {
   static constexpr const char* name = "VOID";
 };
 
+template <>
+struct JITTypeTraits<BOOL> {
+  using NativeType = bool;
+  static constexpr bool isFixedWidth = true;
+  static constexpr uint64_t width = sizeof(NativeType);
+  static constexpr uint64_t bits = sizeof(NativeType) * 8;
```

Comment:
any chance bits != width * 8?

Reply:
I there is no such chance, at least for x86.